demisto_sdk/commands/common/tests/base_validator_test.py

```diff
@@ -55,6 +55,34 @@ def test_handle_error():
     assert 'path/to/file_name - [ST109]' in FOUND_FILES_AND_ERRORS
 
 
+def test_handle_error_file_with_path():
+    """
+    Given
+    - An ignore errors list associated with a file_path.
+    - An error, message, code and file paths.
+
+    When
+    - Running handle_error method.
+
+    Then
+    - Ensure the resulting error messages are correctly formatted.
+    - Ensure ignored error codes return None.
+    - Ensure non ignored errors are in FOUND_FILES_AND_ERRORS list.
+    - Ensure ignored error are not in FOUND_FILES_AND_ERRORS and in FOUND_FILES_AND_IGNORED_ERRORS
+    """
+    base_validator = BaseValidator(ignored_errors={"file_name": ["BA101"], "path/to/file_name_with_path": ["ST109"]},
```

Comment:
currently your test doesn't check the logic you've added (get_relative_path_from_packs_dir won't do anything, because the file paths are not real).
please use TestSuite to make the test check the exact use case explained in the issue.